backend/cabin-sorting/controllers/sortedHackers-controller.ts

```diff
@@ -14,4 +14,10 @@ const createSortedHacker = async (req: any, res: any) => {
   return createResponse;
 };
 
-export default {getSortedHackers, createSortedHacker};
\ No newline at end of file
+const getGroupedHackers = async (_req: any, res: any) => {
+  const groupedHackers = await sortedHackersService.getGroupedHackers()
```

Comment:
I think we should wrap everything in a try, catch block and also check if the result is empty/null